classes/Repository/AttributesRepository.php

```diff
@@ -47,14 +47,16 @@ public function getAllAttributes(): array
 
         foreach ($customAttributes as $attr) {
             $attributes[] = [
-                'name' => strtolower($attr['name']),
+                // Not the best way in terms of permances, but avoid being responsible of a whole SQL query.
+                'name' => (new \AttributeGroupCore($attr['id_attribute_group']))->name,
```

Comment:
Can you use strtolower function? it's better for the front to displaying

Reply:
Actually I was wondering why we had this in the first place. In the list displayed to the merchant, we have the common fields that are properly displayed (With the first letter in upper case), but the custom ones are displayed like we have only a technical name.
This change makes the content more homogenous, or at least with the original values

Reply:
Okay no problem I understand the reasoning and I agree with you